tests/components/automation/test_init.py

```diff
@@ -2140,12 +2140,12 @@ async def async_automation_triggered(event):
         hass.bus.async_listen("automation_triggered", async_automation_triggered)
 
         hass.bus.async_fire("trigger_automation")
-        await asyncio.wait_for(script_done_event.wait(), 1)
+        await asyncio.wait_for(script_done_event.wait(), 10)
```

Comment:
We have been trying to get rid of all the wait_for and replace than with async_timeout or asyncio.timeout with py311

Reply:
there are a large amount of these wait 1 seconds stuff in our tests for scripts. so they are sensitive to test server load. So I think just replacing with async_timeout is not going to help. I've triggered this timeout once at least.

Reply:
Increasing the timeout is fine but 10s it probably too long since the runner times out at 9 if I remember correctly.

async_timeout is just a bit more efficient since it doesn’t create a task.  From a practical sense it doesn’t matter here.  The goal is to get rid of its usage from the codebase so we set an example that async_timeout is preferred since we had lots of places where the additional task being created did matter

Reply:
Since we have such a short runner timeout, we should just remove these wait_for fully maybe. I suppose some performance limit does make sense.